src/Verification.php

```diff
@@ -79,37 +133,77 @@ private function expectedSignature()
      */
     private function providedSignatureBase64()
     {
-        return $this->parameter('signature');
+        return $this->signatureHeaderParameter('signature');
+    }
+
+    /**
+     * @return string
+     *
+     * @throws Exception
+     */
+    private function providedAuthorizationBase64()
+    {
+        return $this->authorizationHeaderParameter('signature');
     }
 
     /**
      * @return Key
      *
      * @throws Exception
      */
-    private function key()
+    private function signatureKeyId()
     {
-        return $this->keyStore->fetch($this->parameter('keyId'));
+        return $this->keyStore->fetch($this->signatureHeaderParameter('keyId'));
     }
 
     /**
-     * @return HmacAlgorithm
+     * @return Key
+     *
+     * @throws Exception
```

Comment:
says `throws Exception` but the method doesn't

Reply:
But the invoked method does (following the chain). Is this hint only for direct throws in this function, or unhandled exceptions in all further called functions within?

Reply:
@mtibben Happy to remove these hints, but I want to be consistent so your advice is appreciated.

Reply:
bump...